python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py

```diff
@@ -142,13 +151,149 @@ def __init__(self,
         self.onnx_available = False
         self.quantize_available = False
         self.use_amp = False
+        self.use_hpo = True
 
-        # seed setting
-        from pytorch_lightning import seed_everything
-        seed_everything(seed=self.seed)
+        has_space = self._config_has_search_space(
+            config={**self.model_config, **self.optim_config,
+                    **self.loss_config, **self.data_config})
+
+        if not self.use_hpo and has_space:
```

Comment:
it seems that this condition will not be passed under any circumstances since `self.use_hpo` is always True.

Reply:
done. remove this check.